src/frame-writer.cpp

```diff
@@ -212,15 +212,62 @@ AVPixelFormat FrameWriter::handle_buffersink_pix_fmt(const AVCodec *codec)
     return best_format;
 }
 
+static std::string transpose_from_transform(int32_t transform)
+{
+    switch (transform)
+    {
+      case WL_OUTPUT_TRANSFORM_90:
+        fprintf(stderr, "Transform: 90\n");
+        return "transpose=" + std::to_string(1);
+        break;
+      case WL_OUTPUT_TRANSFORM_180:
+        fprintf(stderr, "Transform: 180\n");
+        return "transpose=" + std::to_string(1) + ",transpose=" + std::to_string(1);
+        break;
+      case WL_OUTPUT_TRANSFORM_270:
+        fprintf(stderr, "Transform: 270\n");
+        return "transpose=" + std::to_string(2);
+        break;
+      case WL_OUTPUT_TRANSFORM_FLIPPED:
+        fprintf(stderr, "Transform: FLIPPED\n");
+        return "hflip";
+        break;
+      case WL_OUTPUT_TRANSFORM_FLIPPED_90:
+        fprintf(stderr, "Transform: FLIPPED_90\n");
+        return "transpose=" + std::to_string(1) + ",hflip";
```

Comment:
I thought the `FLIPPED_x` transforms mean first hflip then the transform? https://wayland.app/protocols/wayland#wl_output:enum:transform